src/main/java/com/uipath/uipathpackage/models/LocalizationOptions.java

```diff
@@ -1,6 +1,6 @@
 package com.uipath.uipathpackage.models;
 
-public abstract class LocalizationOptions {
+public abstract class LocalizationOptions extends TelemetryOptions {
```

Comment:
Maybe let's keep this separate, Localization and telemetry are different. I understand that you want to extend the only class and that's why you have added it. But we can extend these two options in a class. or create a combined class which can be extended by the class

Reply:
Done